heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/filter/helper/HelperImpl.java

```diff
@@ -81,11 +80,10 @@ public DB db(String databaseName) {
 	}
 
 	private DB db(String databaseName, DBType type) {
-
-		switch (type) {
-		default:
+		if(type.equals(DBType.MONGODB)){
```

Comment:
`switch-default` is always executed is this context, so, this if is unnecessary.